Sources/MuxPlayerSwift/PublicAPI/Extensions/AVPlayerItem+Mux.swift

```diff
@@ -83,6 +83,24 @@ fileprivate func makePlaybackURL(
     return playbackURL
 }
 
+/// Create a new `AVAsset` that has been preparted for playback
+/// Currently, being "prepared for playback" means only that the DRM token was added to the asset's options if present
+fileprivate func makeAVAsset(playbackID: String, playbackOptions: PlaybackOptions) -> AVAsset {
+    let url = makePlaybackURL(playbackID: playbackID, playbackOptions: playbackOptions)
+    
+    var asset: AVURLAsset
+    if case .drm(let options) = playbackOptions.playbackPolicy {
+        PlayerSDK.shared.fairplaySessionManager.registerDrmToken(options.drmToken, for: playbackID)
+        asset = AVURLAsset(url: url)
```

Comment:
`AVURLAsset(url: url)` can be moved outside of the `if case let` since it's the same in both the if and else clauses. The `else` clause can then get dropped entirely.